diff --unified --recursive buildSrc/src/main/groovy/com/sun/javafx/gradle/NativeCompileTask.groovy buildSrc/src/main/groovy/com/sun/javafx/gradle/NativeCompileTask.groovy
--- buildSrc/src/main/groovy/com/sun/javafx/gradle/NativeCompileTask.groovy	2019-12-30 02:55:03.931075684 -0500
+++ buildSrc/src/main/groovy/com/sun/javafx/gradle/NativeCompileTask.groovy	2019-12-30 02:55:15.415961722 -0500
@@ -102,7 +102,7 @@
         updateFiles();
         def source = project.files(allFiles);
         boolean forceCompile = false;
-        final Set<File> files = new HashSet<File>();
+        Set<File> files = new HashSet<File>();
         source.each { File file ->
             final Map fileData = dependencies.get(file.toString());
             final boolean isModified = fileData == null ||
